app/models/services/AuthTokenServiceImpl.scala

```diff
@@ -0,0 +1,57 @@
+package models.services
+
+import java.sql.Timestamp
+import java.time.Instant
+import java.util.UUID
+import javax.inject.Inject
+
+import com.mohiva.play.silhouette.api.util.Clock
+import models.AuthToken
+import models.daos.AuthTokenDAO
+import org.joda.time.DateTimeZone
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import play.api.libs.concurrent.Execution.Implicits._
+
+/**
+ * Handles actions to auth tokens.
+ *
+ * @param authTokenDAO The auth token DAO implementation.
+ * @param clock        The clock instance.
```

Comment:
Are either of these parameters used in this file?

Reply:
Nope, I think I left those over